src/main/java/com/example/HumanPlayerFactory.java

```diff
@@ -0,0 +1,41 @@
+package com.example;
+
+import java.util.Scanner;
+
+public class HumanPlayerFactory implements PlayerFactory
```

Comment:
I think there is no need for human factory,
As there are not different kind of humans,
Like extra configuration and all.